lib/ui/cards/student_info/student_info_card.dart

```diff
@@ -91,6 +113,16 @@ class _StudentInfoCardState extends State<StudentInfoCard> {
   }
 
   void reloadWebView() {
-    _webViewController?.reload();
+    if(Theme.of(context).brightness == Brightness.dark) {
```

Comment:
Can you make a theme helper function and change all these usages to use the helper?

Reply:
Sure, I'll work on that later today. Thanks for the suggestion, it will definitely make the code more readable and maintainable.